### DISCUSSION TITLE
PacBio and Illumina FASTQ Files

### DISCUSSION THREAD

**Discussion by kevinmyers** [wwood/CoverM#120] [Q&A] [answered]
I have a set of 217 MAGs that I would like to use coverM to determine the overall coverage of a number of FASTQ files. For some of our experiments, we have both Illumina and PacBio sequencing files. Should I run CoverM on the Illumina and PacBio sequencing files separately or is it okay to run everything together?

I'm relatively new to CoverM and this kind of thing. Our goal is to determine relative abundance of the MAGs across different experiments, if that helps.

**Comment by wwood** [comment, marked as answer]
Hi @kevinmyers this is a good question.

You can carry out the mapping based part of it using `coverm genome --bam-file-cache-directory` using `-p minimap2-pb` and `-p minimap2-sr`. This will generate the BAM files. Then merge those BAM files, and then run `coverm genome` again using the merged BAM file as input.

It's a bit messy I know - the problem is that currently there is no way to specify multiple `-p` in a single command (an no way to group multiple readsets into a single sample). I'll take this as a feature request then, especially since hybrid sequencing is more common these days.

**Reply by kevinmyers** [reply]
Thanks! Just so I understand, I should run `coverm` two times, once on the PacBio samples with `-p minimap2-pb` and once on the Illumina samples with `-p minimap2-sr`. And by merge BAMs, you mean concatenate the BAM files or just run `coverm` again using all the BAMs as input?

**Reply by wwood** [reply]
Right

**Reply by kevinmyers** [reply]
Thanks! I'll do that then run the BAMs together. Appreciate the quick replies!

**Reply by wwood** [reply]
Also, just to be clear, input BAM files need to be sorted.

**Reply by kevinmyers** [reply]
Thanks! Will do!
